root/etc/httpd/conf.d/janus-admin-api.conf

```diff
@@ -0,0 +1,5 @@
+Alias "/janus-admin-api" "/opt/janus/share/janus/demos"
```

Comment:
This configuration opens a demo software to the public world, I don't think it's safe enough.
99% of the users will not use it and event will not be aware to have it publicly accessible.

Since this is a simple drop-in useful only for developers, I'd move it to the README.

Reply:
Yes, you are right, I thought about that.
All the exposed demo require 8088 port to be open, but it is closed by default.
But I can agree that there is some code not useful for us that can expose some security aspects.
So, if you agree I can remove all the demos by removing httpd configuration and then developers can use admin api by curl, is it ok ?

Reply:
Yes, it's ok for me! ;)

Reply:
Done :+1: